bodo/tests/test_sql.py

```diff
@@ -24,7 +24,7 @@
 )
 from bodo.utils.testing import ensure_clean_mysql_psql_table
 
-pytestmark = pytest.mark.sql
+pytestmark = [pytest.mark.sql, pytest.mark.slow]
```

Comment:
Similarly, this feature is mostly broken and not often updated so marked as slow. If you have any suggestions for other tests to mark as slow, let me know.